dmriprep/utils/vectors.py

```diff
@@ -142,14 +162,50 @@ def generate_rasb(self):
             _ras = bvecs2ras(self.affine, self.bvecs)
             self.gradients = np.hstack((_ras, self.bvals[..., np.newaxis]))
 
+    def reorient_rasb(self):
+        """Reorient the vectors based o a list of affine transforms."""
+        from dipy.core.gradients import gradient_table_from_bvals_bvecs, reorient_bvecs
```

Comment:
Why not import everything at the top of the module? Is this common practice?

Reply:
@arokem -- I'd be happy to modify this to a top-level import if everyone thinks that'd be more appropriate. I chose to bake it into the method initially since I think of it like a situational import -- on most occasions of using the DiffusionGradientTable interface, the `reorient_rasb` method won't be called.